packages/app-store/zohocalendar/types/ZohoCalendar.ts

```diff
@@ -0,0 +1,42 @@
+export type ZohoAuthCredentials = {
```

Comment:
@hariombalhara 
Are we using the `types` directory in new apps?

Reply:
I think it might be a good idea. Depending on the complexity, I think apps can decide to add their own dirs. So, I am fine with it.